src/ctia/http/routes/common.clj

```diff
@@ -25,13 +25,18 @@
 (def filter-map-search-options
   (conj search-options :query :simple_query :from :to))
 
+(s/defschema DateRangeParams
+  (st/optional-keys
+    {:from s/Inst
+     :to s/Inst}))
+
 (s/defschema BaseEntityFilterParams
-  {(s/optional-key :id) s/Str
-   (s/optional-key :from) s/Inst
-   (s/optional-key :to) s/Inst
-   (s/optional-key :revision) s/Int
-   (s/optional-key :language) s/Str
-   (s/optional-key :tlp) s/Str})
+  (st/merge
+   DateRangeParams
+   {(s/optional-key :id) s/Str
```

Comment:
How about wrapping the entire map with `st/optional-keys`? Same as for DateRangeParams

Reply:
`DateRangeParams` already wraps the constituents with `st/optional-keys`.